devices/AtomQrCode/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CSharp.TestApplication")]
```

Comment:
Please copy/paste from other sample to have some coherence and a nice naming as well, a lot of the rest if not needed